src/main/java/pl/edu/pw/elka/pik/entities/Achievement.java

```diff
@@ -0,0 +1,54 @@
+package pl.edu.pw.elka.pik.entities;
+
+import javax.persistence.*;
+
+/**
+ * Created by Mateusz on 2016-05-31.
+ */
+@Entity
+public class Achievement {
```

Comment:
The achievements are implemented by the more generic concept of tags. This new table is not necessary.